modules/dev-tools/src/helpers/esm-alias.ts

```diff
@@ -6,16 +6,29 @@
 import path from 'path';
 import fs from 'fs';
 import {pathToFileURL} from 'url';
-import {resolve as resolveTs, getFormat, transformSource, load} from 'ts-node/esm';
 import {getValidPath, ocularRoot} from '../utils/utils.js';
-export {getFormat, transformSource, load};
+
+type ResolveHook = (
```

Comment:
Nit: Add TSDoc on the type (and fields if possible). Specify if this is this an ocular type or are we compensating for esbuild 's lack of types, if so which version of esbuild was this written for?

Reply:
This is Node.js API. I'll add a comment.